script/bootstrap

```diff
@@ -36,10 +36,13 @@ if (process.platform === 'win32') deleteMsbuildFromPath()
 
 installScriptDependencies(ci)
 installApm(ci)
+const apmVersionEnv = Object.assign({}, process.env);
+// Set resource path so that apm can load Atom's version.
+apmVersionEnv.ATOM_RESOURCE_PATH = CONFIG.repositoryRootPath;
```

Comment:
Less readable. I prefer not. Just a personal preference.

To Atom's official maintainers: whatever way of formatting this you prefer, I don't mind.

Reply:
> Less readable. I prefer not. Just a personal preference.

Really? It is interesting. 

Many people prefer the simple "object definition" syntax (`{key: value}`), instead of using JavaScript API directly (`Objecy.assign({}, obj)`, etc). Here the performance is not important but in a general case, my version will be faster.

Reply:
I'm not a JS expert. I find the code I wrote more readable, FWIW.

Note: I copy-pasted this from `script/lib/run-apm-install` and just changed the variable name for the new `env` object. If the syntax you give is really better, it should be consistent in both files.

You have provided some justification. I consider it between you and Atom's maintainers now.

Reply:
Consistency is better, because somebody is going to get tripped up reading this some time in the future.

In both places we are doing the exact same steps: Creating a new `env` object and setting `ATOM_RESOURCE_PATH` to `CONFIG.repositoryRootPath`.

It's the exact same operations, it should use the same code in both places. I'm not interested in doing that unless maintainers here tell me to.

If the different syntax is important to you, and if it's not uncluded in this PR before merging, please do it in a follow-up PR. Like I said, it's not happening in this PR unless the official maintainers ask me to.

I think that covers this topic, please wait for maintainers to respond or for them to merge this PR. I've run out of things to add. Thanks.

Reply:
Personally, I don't think this is necessary, but If you prefer to force a certain style of programming, you use an Eslint plugin. There is one for object spread already:
https://www.npmjs.com/package/eslint-plugin-prefer-object-spread